packages/tasit-action/src/contract/Action.js

```diff
@@ -77,7 +77,10 @@ export class Action extends Subscription {
     const baseEthersListener = async blockNumber => {
       try {
         if (!this.#tx) this.#tx = await this.#txPromise;
-
+        
+        if (this.#tx.status == 0) {
```

Comment:
Hi @Olivierlu1 , we are trying to use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) whenever is possible, fell free to do that with `this.#tx` too.

Reply:
Hey @Olivierlu1, I've just realized that destructuring syntax isn't works with https://github.com/tc39/proposal-class-fields private fields.
So, the syntax `const { #tx } = this;` won't work.

Reply:
@marcelomorgado Good catch. I'd vote for moving to `const tx = this.#tx`